client_code/ProgressBar/Determinate/__init__.py

```diff
@@ -33,6 +35,24 @@ def progress(self, value):
         self._props["progress"] = value
         self.indicator_dom_node.style.setProperty("width", f"{value:%}")
 
+    @property
+    def height(self):
+        return self._height
+
+    @height.setter
+    def height(self, value):
+        self._height = value
+        self.indicator_dom_node.style.setProperty("height", value)
+
+    @property
+    def width(self):
+        return self._width
+
+    @width.setter
+    def width(self, value):
+        self._width = value
+        self.dom_node.style.setProperty("width", value)
```

Comment:
There's a utility function for this so we can support number as well as strings. 

It's in _component_helpers, _css_length.

Reply:
Perfect, that makes sense. I did modify that utility function so that strings with the unit already applied can be used as well.